docs/DeveloperGuide.md

```diff
@@ -294,17 +294,71 @@ The activity diagram shows the general sequence of steps when a user interacts w
 
 <br>
 
+## Implementation of Entity Commands
+Entity commands include `add`, `delete`, `find`, `clear` commands
+
+**Step 1**. The user types an `xyz` command in the `CommandBox`, followed by the type of entity `person` or `appt`, followed by appropriate arguments and prefixes.
+**Step 2**. The command is passed to the `LogicManager`. `LogicManager` then calls the `AddressBookParser::parseCommand` method to parse the command
+**Step 3**. The `AddressBookParser` then creates a `xyzEntityCommand` by passing string to `xyzCommandParser` and is returned to the `LogicManager`.
```

Comment:
Would it be clearer if we break this step down? It would be easier to explain with a sequence diagram as well!

Eg 
**Step 3.** The `AddressBookParser` creates an `xyzCommandParser` object, and calls the `xyzCommandParser::parse` method. 
**Step 4.** The `xyzCommandParser` creates an `xyzCommand`, which is returned to the `LogicManager`.
...

Reply:
Should we also give examples of what `xyzCommandParser` and `xyzCommand1` is? Eg `AddCommandParser`, `AddCommand`...